configs/warp-route-token-config.json

```diff
@@ -1,15 +1,15 @@
 {
   "goerli": {
     "type": "collateral",
-    "handleGasOverhead": 68000,
```

Comment:
Can we default this in the deployer instead? (and allow overrides, similar to the ism)